### DISCUSSION TITLE
Ny karta

### DISCUSSION THREAD

**Discussion by jokd** [origo-map/origo#1592] [General]
Hur får man igång en Origokarta?

**Comment by filleg** [comment]
> Hur får man igång en Origokarta?

Följer dokumentationen!

**Reply by jokd** [reply]
Tack!

**Reply by Grammostola** [reply]
Vad jag ser att vi har idag i readme.md (som borde vara first contact) är:

## How to get started

Download the latest [release](https://github.com/origo-map/origo/releases/) and check out the [documentation](https://origo-map.github.io/origo-documentation/latest/#origo-api). 

Säger inte så mycket om build/ eller var göra av den i en typisk installation. Dessa ting nämns under "Getting started" i samma readme.md som dock börjar "So you have decided to take matters into your own hands and get coding? Great!" vilket kanske inte är vad man vill get started med.

Dokumentationen har inga installations/driftsättningsanvisningar vad jag ser.

Edit. Det kanske inte är det frågeställningen gäller, jag spekulerar : )

**Comment by sonjahalmstad** [comment]
Det är lite klurigt kan man tycka...

**Reply by matself** [reply]
Klurigt är bara förnamnet. Jag har letat en lång stund på GitHub nu efter installations- och driftsättningsanvisningar men inte hittat en enda rad. Jag har inte letat innan, bara tagit för givet att det ska finnas. Hur kommer det sig att det saknas?
Helst skulle jag vilja köra Origo på min Ubuntu-server men det tycks som att det är en Windowsapplikation enbart?

**Reply by Grammostola** [reply]
Det är en webbapplikation som kan köras av vilken webbserver som helst, ex IIS på Windows eller Apache i en Linux-miljö. Vi skulle kunna ha något om var göra om build/ i en eller ett par populära OS+webbservrar? Eller bara en kort mening ditåt.
